mastercraft.cmd

```diff
@@ -2049,6 +2113,7 @@ lack.material:
                if "%order.type" = "genich" then var order.num 11
                if "%order.type" = "ojhenik" then var order.num 12
                if "%order.type" = "red.flowers" then var order.num 13
+			   if "%order.type" = "flowers" then var order.num 13

```

Comment:
Added handling for flowers instead of red flowers. When you buy them they are red flowers, when you combine them they are dried flowers.